server/@types/journeys/index.d.ts

```diff
@@ -10,15 +10,16 @@ export interface PrisonerDetails {
 }
 
 export type Address = {
-  id: string
+  id?: string | null
+  description?: string | null
   flat?: string | null
```

Comment:
Guessing this could all be removed but you're keeping it until we're sure we're not going back? I'm pretty confident we aren't now